src/NLog.DiagnosticSource/NLog.DiagnosticSource.csproj

```diff
@@ -14,19 +14,21 @@
 
     <Title>NLog.DiagnosticSource</Title>
     <Company>NLog</Company>
-    <Description>NLog Integration for Microsoft System.Diagnostics.DiagnosticSource and Activity </Description>
+    <Description>NLog Integration for Microsoft's System.Diagnostics.DiagnosticSource and Activity</Description>
     <RepositoryType>git</RepositoryType>
     <RepositoryUrl>https://github.com/NLog/NLog.DiagnosticSource.git</RepositoryUrl>
     <PackageTags>DiagnosticSource;NLog;logging;log</PackageTags>
     <PackageIconUrl>https://nlog-project.org/N.png</PackageIconUrl>
     <PackageProjectUrl>https://github.com/NLog/NLog.DiagnosticSourcer</PackageProjectUrl>

```

Comment:
Think also it should be https://github.com/NLog/NLog.DiagnosticSource without an "r" at the end